doc/rados/operations/placement-groups.rst

```diff
@@ -132,10 +132,10 @@ The output will resemble the following::
   pool is working towards. 
 
 - **NEW PG_NUM** (if present) is the value that the system is recommending the
-  ``pg_num`` of the pool to be changed to. It is always a power of 2, and it is
-  present only if the recommended value varies from the current value by more
-  than the default factor of ``3``. To adjust this factor (in the following
-  example, it is changed to ``2``), run the following command:
+  ``pg_num`` of the pool to be changed to. It is always a power of two, and it
```

Comment:
is recommending for the pool.  It is

Reply:
I'm chary of for-phrases, because they're so hard for non-native English speakers to interpret. However, I do see how the end of this sentence is a bit of a humdinger. So I will tweak it  little, and hope that the tweaking makes it a little easier to read.